New entry: HTML `<noscript>` tag (#781)
Codecademy/docs: content/html/concepts/tags/terms/noscript/noscript.md

---
Title: '<noscript>'

> Description: 'Displays if the user disabled Javascript in the browser.'

Reviewer: ```suggestion
Description: 'Displays content within if Javascript is disabled in the browser or not supported.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<noscript>` tag will display its contents when the user disabled Javascript for that site.

Reviewer: ```suggestion
The `<noscript>` tag displays its contents whenever [Javascript](https://www.codecademy.com/resources/docs/javascript) is disabled on the browser or it is not supported for a given site.
```

  
## Syntax
  
```html
<noscript>

>   <!-- What should display if JS is not enabled -->

Reviewer: ```suggestion
  <!-- What should display if JavaScript is not enabled or supported. -->
```

>   

Reviewer: ```suggestion

```

>   

Reviewer: ```suggestion

```

</noscript>
```

> The tag has no attributes and what is nested within the tag will be displayed (if JS is not enabled).

Reviewer: ```suggestion
The tag has no attributes and what is nested within the tag will be displayed.
```

  
## Example

```html

> <script href="someScript.js"></script>

Reviewer: What is the purpose of this `<script>` tag being here?

Author: @Dusch4593 Why would you use the no script tag if you don’t have JavaScript?

Reviewer: I'm saying that if the `<script>` tag plays a role in this example, it should be explained in text-form above the snippet (but below the "Example" heading).

Also, `href` is not a valid attribute with the `<script>` tag; use `src="someScript.js"`, instead.

> <no-script>

Reviewer: ```suggestion
<noscript>
```

  <p>Please enable Javascript for this page to load. Learn how to <a href="https://support.google.com/adsense/answer/12654?hl=en">here</a></p>
</noscript>
```
  

> Output of the example if JS is not enabled:

Reviewer: ```suggestion
The output might look like this:
```

>   
> The output might look like this:
>   

Reviewer: ```suggestion

The output might look like this:

```
